djangocms_content_expiry/monkeypatch/admin.py

```diff
@@ -0,0 +1,32 @@
+from django.utils.translation import ugettext_lazy as _
+
+from djangocms_versioning import admin
+
+from djangocms_content_expiry.models import ContentExpiry
+
+
+def expires(self, obj):
+    version = ContentExpiry.objects.filter(version=obj.pk)
+    if version:
+        return version[0].expires
+    return ""
+
+
+expires.short_description = _('expire date')
+admin.VersionAdmin.expire = expires
+
+
+def get_list_display(func):
+    """
+    Register the locked field with the Versioning Admin
+    """
+
+    def inner(self, request):
+        list_display = func(self, request)
+        created_by_index = list_display.index('created_by')
+        return list_display[:created_by_index] + ('expire',) + list_display[created_by_index:]
```

Comment:
Just noticed that this isn't translated. 
```suggestion
        return list_display[:created_by_index] + (_('expire'),) + list_display[created_by_index:]
```